.travis.yml

```diff
@@ -0,0 +1,18 @@
+---
+language: go
+
+go:
+  - "1.10.x"
+
+before_install:
+  - export GOPATH="${HOME}/go:$GOPATH"
+  - export PATH="${HOME}/go/bin:$PATH"
+  - sudo add-apt-repository ppa:jonathonf/python-3.6
+  - sudo apt-get update && sudo apt-get install python3.6
+  - pip3 install ipaddr
+  - sudo apt-get install software-properties-common
+  - sudo apt-add-repository --yes --update ppa:ansible/ansible
+  - sudo apt-get install ansible
+install:
+  - cd {TRAVIS_BUILD_DIR}/ansible
+  - ansible-playbook main.yml --extra-vars "ip=35.190.178.102 reset_network=true"
```

Comment:
Variables like this ip address should be passed via command line or through a separate parameter file

Reply:
I don't think you are looking at the latest changes